### ISSUE TITLE
Type annotations / type hints / stub files for mypy

### ISSUE THREAD

**Issue by MartinThoma** [sh4nks/flask-caching#197]
Are there plans to add type annotations or stub files for mypy?

I've written an [article about type annotations](https://medium.com/analytics-vidhya/type-annotations-in-python-3-8-3b401384403d) if you need one :-)

**Comment by sh4nks** [comment]
I have no plans implementing this feature myself, however I'd gladly review a PR for it!

**Comment by MartinThoma** [comment]
@sh4nks Did you have the time to have a look at my pr?

**Comment by sh4nks** [comment]
Included in v1.10. Thanks for your contribution!

**Comment by hgranthorner** [comment]
NOTE: I'm a new at mypy so this may be an ID10T error.

I just updated my project from ~1.7 to 1.10.1 but mypy is not recognizing any built in types for flask-caching. It is still reporting this error:
```
Skipping analyzing 'flask_caching': found module but no type hints or library stubs
```
Can anyone else reproduce this or is there something wrong with my set up?
